src/js/lineChartTimeSeriesMultiDataSet.js

```diff
@@ -0,0 +1,184 @@
+        model: {
+            dataSet: [],
+            // See lineTests.js for the style of the dataSets that are
+            // allowed
+            svgTitle: "Line Chart",
+            svgDescription: "A line chart."
+        },
+        bindings: {
+            title: "svgTitle",
+            description: "svgDescription"
+        },
+        svgOptions: {
+            width: 700,
+            height: 500
+        },
+        lineOptions: {
+            // An array of colors to use for lines drawn from the data
+            // Or, a d3 color scale that's generated based off an array of colors
+            colors: null,
+            // Padding of the chart elements inside the SVG container
+            padding: 50,
+            // interpolation mode for chart lines and areas
+            // see line.interpolate at https://github.com/mbostock/d3/wiki/SVG-Shapes
+            // generally, "linear" for sharp lines, "cardinal" for smooth, or "step" for a step chart
+            interpolation: "linear",
+            // In milliseconds
+            transitionLength: 2000
+        },
+        scaleOptions: {
+            // transform rules to apply to yScale min
+            yScaleMinTransform: {
+                "literalValue": 0
+            },
+            // transform rules to apply to yScale max
+            yScaleMaxTransform: {
+                "transform": {
+                    "type": "fluid.transforms.binaryOp",
+                    "leftPath": "max",
+                    "right": 1.25,
+                    operator: "*"
+                }
+            }
+        },
+        events: {
+            onChartCreated: null,  // Fire when the line is created. Ready to register D3 DOM event listeners,
+            onDraw: null
+        },
+        listeners: {
+            "onCreate.create": {
+                funcName: "floe.chartAuthoring.lineChart.timeSeriesMultiDataSet.create",
+                args: ["{that}"]
+            }
+        },
+        modelListeners: {
+            dataSet: [{
+                funcName: "{that}.events.onDraw.fire",
+                excludeSource: "init"
+            }]
+        },
+        invokers: {
+            getXScale: {
+                funcName: "floe.chartAuthoring.lineChart.timeSeriesMultiDataSet.getXScaleTimeSeries",
+                args: ["{that}"]
+            },
+            getYScale: {
+                funcName: "floe.chartAuthoring.lineChart.timeSeriesMultiDataSet.getYScale",
+                args: ["{that}"]
+            }
+        }
+    });
+
+    floe.chartAuthoring.lineChart.timeSeriesMultiDataSet.fixedValue = function (fixedValue) {
+        return fixedValue;
+    };
+
+    floe.chartAuthoring.lineChart.timeSeriesMultiDataSet.multiplyValue = function (value, multiplier) {
```

Comment:
Yup - old functions from before I figured out how to actually use transforms. Removed.